### ISSUE TITLE
[Metricbeat] Add new metrics for swapin/out

### ISSUE THREAD

**Issue by radoondas** [elastic/beats#13175]
**Describe the enhancement:**
Hi, I would like to be able to collect metrics from swapin/out as showed below.

```
root@server42:/etc/systemd/system# cat /proc/vmstat|grep swp
pswpin 399224
pswpout 1291234
```
Thanks

**Comment by fearful-symmetry** [comment]
Looking into this. On linux, the current swap stat implementation used by `elastic/gosigar` uses `/proc/meminfo` and not `/proc/vmstat`. However, it shouldn't be hard to grab the vmstat metrics and add them to the (limited) swap data we're getting from meminfo.

**Comment by fearful-symmetry** [comment]
So, oddly enough, I don't see anything in gosigar that reads from the vmstat procfs file under linux. Gonna have to start there.

**Comment by fearful-symmetry** [comment]
Okay, finally starting work on this. It looks like we'll be using the elastic/go-sysinfo library.

**Comment by kkh-security-distractions** [comment]
Greta to hear, that this is underway There could be other useful stuff i /proc/vmstat , but swap info is key for us right now. Let us if you want some beta testers , happy to help out.

**Comment by fearful-symmetry** [comment]
This is merged, and should be in 7.5

**Comment by kkh-security-distractions** [comment]
awesome ,thnx
